.github/ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,8 @@ This is a (multiple allowed):
 * [ ] feature-discussion (RFC)
 
 ## DESCRIPTIVE TITLE HERE
```

Comment:
This can be removed I think. People generally fill out the title of the issue well enough.